src/coreComponents/dataRepository/Group.hpp

```diff
@@ -863,6 +865,16 @@ class Group
   ///@}
   //END_SPHINX_INCLUDE_REGISTER_WRAPPER
 
+  /**
+   * @brief Append a levelCondition and a log description to the description of the wrapped object given a log info struct.
+   * Must be called in constructor.
+   * @tparam LOG_LEVEL_INFO The log documentation to add.
+   * @return void if the trait is verified.
+   */
+  template< typename LOG_LEVEL_INFO >
+  std::enable_if_t< geos::is_log_level_info< LOG_LEVEL_INFO >, void >
+  addLogLevel();
```

Comment:
does it actually add a `LogLevel` or is it rather adding to a `LogLevel`? We can discuss this further and leave it for a follow up PR but I wonder if the possible levels should not be fixed somewhere and be the same for all groups and let this function add logging to one of the predefined options. Although I do know that some times unusual log level numbers are added for Hypre.

Reply:
It add a `logLevel` wrapper if there is none, and the current `loglevel` description is appended to the wrapper description, all subClasses inherit descriptions from parent; if that answers the 1st part of your question.